src/soda-js.coffee

```diff
@@ -87,7 +87,14 @@ expr =
   lt:  (column, literal) -> "#{column} < #{handleLiteral(literal)}"
   lte: (column, literal) -> "#{column} <= #{handleLiteral(literal)}"
   eq:  (column, literal) -> "#{column} = #{handleLiteral(literal)}"
-
+  
+# serialize object to querystring
+serialize = (obj) ->
```

Comment:
Might be helpful to name this function a little more specifically, since it's not a general serializer.